src/main/java/com/automation_boss/app/App.java

```diff
@@ -39,5 +45,47 @@ public static void main(String args[]) throws IOException, InterruptedException
         r1.join(firstAttendee);
         r1.join(secondAttendee);
         r1.describeInventory(myFile);
+        System.out.println("room Chair comparison");
+        if (roomChair.equals(roomChair2))
+            System.out.println("true - " + roomChair.toString() + " - " + roomChair2.toString());
```

Comment:
`.toString()` will be invoked automatically in `System.out.println`. That's why you can simplify the code like `System.out.println("true - " + roomChair + " - " + roomChair2);`.

Please update all lines below.